app/assets/javascripts/restrictFeatures.js

```diff
@@ -0,0 +1,72 @@
+var feature_restriction_metadata = {
```

Comment:
Use stimulas for ui interactions https://stimulus.hotwired.dev/

Reply:
Sure